frontend/rolecall/src/app/cast/cast-editor-v2.component.ts

```diff
@@ -55,8 +63,8 @@ export class CastEditorV2 implements OnInit {
     }
   }
 
-  onSelectPiece(piece: Piece) {
-    this.setPiece(piece);
+  onSelectPiece(pieceIx: number) {
```

Comment:
Would be nice to just call this `selectPiece`, doesn't need the "on" prefix

Reply:
OK.